tests/foreman/cli/test_gpgkey.py

```diff
@@ -135,9 +135,7 @@ def test_positive_create_with_default_org(self):
 
         @assert: gpg key is created
         """
-        result = Org.list()
-        self.assertGreater(len(result), 0, 'No organization found')
```

Comment:
`.list()` may return empty list if command succeeded but stdout is empty (no results found), so assertion after list command is a good thing, `.info()` will have non-zero return-code in case of entity not found, and as we catch return-codes of all the executed commands, if org wasn't found we'll raise an exception here.
This change is good as for me as both ways we won't miss failure but this new way we are working with default org directly and not assuming that the first entity returned by `.list()` command is the default org (which is correct right now but no guarranties it won't be changed in future - for some commands the list is sorted alphabetically by name, not id).